playbooks/support.md

```diff
@@ -31,6 +31,9 @@ hours, engineers are _only_ responsible for downtime issues.
 The two engineers who are on-call are free to split the responsibility for monitoring outside of work hours, but we
 suggest either doing every other day or first half/second half.
 
+### A note about team members who leave
+If someone who is scheduled to be on-call ends up leaving the engineering team (by moving to a different team or leaving Artsy altogether), it is their (previous) manager's responsibility to cover their shift. The manager will either take the shift themselves or, if they are unavailable, find someone who can.
```

Comment:
This looks accurate but is a special case so should maybe fall below the important stuff. You could add a "special cases" section at the end and maybe mention the holiday rotation schedule as well.